packages/utils/src/carequality/compare-orgs-cq.ts

```diff
@@ -0,0 +1,184 @@
+import { getCqCommand as getCqCommandForOrganization } from "../../../api/src/external/carequality/command/create-or-update-organization";
+
+dayjs.extend(duration);
+
+/**
+ * Compare Metriport generated organizations with the respective entries in the CareQuality directory.
+ *
+ * Stores both in the runs/compare-orgs-cq/run_<date>/ directory for further inspection.
+ *
+ * Note: you might need to update the files below on the API package, making the `Request` type `any`
+ * because we're importing directly from that package but ts-node doesn't recognize the @types declaration
+ * on the API's `tsconfig.json` file, even if we add that to Util's `tsconfig`:
+ * - src/routes/auth.ts
+ * - src/routes/util.ts
+ *
+ * To use it:
+ * - Set the required env vars in .env
+ *   - Also, see makeCarequalityManagementAPIFhir for required env vars.
+ * - Run with `ts-node compare-orgs-cq.ts`
+ */
+
+const cxIds: string[] = [];
+
+const apiUrl = getEnvVarOrFail("API_URL");
+export const apiOssProxyInternal = axios.create({ baseURL: `${apiUrl}/internal` });
+
+const getFolderName = buildGetDirPathInside(`compare-orgs-cq`);
+
+export async function main() {
+  await sleep(50); // just to make sure the logs are in not mixed up with Node's and other libs'
+  const startedAt = Date.now();
+  console.log(`########################## Started at ${new Date(startedAt).toISOString()}`);
+
+  const cqApi = makeCarequalityManagementAPI();
+  if (!cqApi) throw new Error("CQ API not initialized");
+
+  initRunsFolder();
+  const outputFolderName = getFolderName("run");
+  fs.mkdirSync(outputFolderName, { recursive: true });
+
+  for (const cxId of cxIds) {
+    const { org, facilities } = await getCxData(cxId);
+    const { log } = out(`${cxId} / ${org.name}`);
+
+    const inputMetriportOrg: Organization = {
+      cxId,
+      id: org.id,
+      organizationNumber: Number(org.oid.split(":")[-1]),
+      type: org.businessType,
+      data: {
+        name: org.name,
+        location: org.location,
+        type: org.type as unknown as OrgType,
+      },
+      oid: org.oid,
+      cqActive: org.cqActive ?? false,
+      cqApproved: org.cqApproved ?? false,
+      cwActive: org.cwActive ?? false,
+      cwApproved: org.cwApproved ?? false,
+      createdAt: new Date(),
+      updatedAt: new Date(),
+      eTag: org.eTag ?? "",
+    };
+
+    if (org.businessType === OrganizationBizType.healthcareProvider) {
+      log(`Provider - Processing org...`);
+      if (!org.cqActive || !org.cqApproved) {
+        log(`Skipping ${org.name} because it is not active or approved in CQ`);
+        continue;
+      }
+      const cqCmd = getCqCommandForOrganization({ org: inputMetriportOrg });
+      const cqOrgDetails = await cmdToCqOrgDetails(cqCmd);
+      const metriportOrg = getOrganizationFhirTemplate(cqOrgDetails);
+      await process("org", org, metriportOrg, outputFolderName, cqApi);
+      // empty line
```

Comment:
cleanup i think

Reply:
Yeah, I added this to help with the readability of the code - if I add a new line, prettier will remove it.

I can remove if you guys think we should not have it, NABD.